src/fragments/lib/auth/ios/signout/10_local_signout.mdx

```diff
@@ -5,13 +5,24 @@
 ```swift
 func signOutLocally() async {
     let result = await Amplify.Auth.signOut()
-    guard let signOutResult = result as? AWSCognitoSignOutResult,
-          case .complete = signOutResult
-    else {
-        print("Sign out failed to complete")
-        return
+    if let signOutResult = result as? AWSCognitoSignOutResult {
+        print("Local signout successful: \(signOutResult.signedOutLocally)")
```

Comment:
I guess the print statement is incorrect.. The print statement won't be true when the `signOutResult` is failed. 

this is our implementation: 

```
public enum AWSCognitoSignOutResult: AuthSignOutResult {

    var signedOutLocally: Bool {
        if case .failed = self {
            return false
        }
        return true
    }
```

Reply:
That would result in the following print statements, right?
```Local signout successful: true``` if it succeeds.
```Local signout successful: false``` if it fails.

That seems fine, but I'm open to other suggestions if you have one.